apps/judicial-system/web/src/routes/Court/InvestigationCase/Confirmation/Confirmation.strings.ts

```diff
@@ -0,0 +1,40 @@
+import { defineMessages } from 'react-intl'
+
+export const confirmation = defineMessages({
+  title: {
+    id: 'judicial.system.court:confirmation.title',
```

Comment:
NB, this is the first time, that I can remember, that we use a non-unique id for these strings. Everything seems to work though.